frontend/src/app/containers/ExperimentPage.js

```diff
@@ -304,36 +304,38 @@ export class ExperimentDetail extends React.Component {
                       }
                       </section>}
                     </div>}
-                    {hasAddon && <section className="stats-section" data-hook="active-user">
-                      <table className="stats"><tbody>
-                        <tr data-hook="version-container">
-                          <td data-l10n-id="version">Version</td>
-                          <td>
-                            <span data-hook="version">{version}</span>
-                            {changelog_url && <span>&nbsp;<a data-l10n-id="changelog" data-hook="changelog-url" href={changelog_url}>changelog</a>, </span>}
-                            <a className="showTour" data-l10n-id="tourLink" onClick={e => this.showTour(e)} href="#">tour</a>
-                          </td>
-                        </tr>
-                        <tr>
-                          <td data-l10n-id="lastUpdate">Last Update</td>
-                          <td data-hook="modified-date">{modified}</td>
-                        </tr>
-                        <tr>
-                          <td data-l10n-id="contribute">Contribute</td>
-                          <td><a data-hook="contribute-url" href={contribute_url}>{contribute_url}</a></td>
-                        </tr>
-
-                        <tr>
-                          <td data-l10n-id="bugReports">Bug Reports</td>
-                          <td><a data-hook="bug-report-url" href={bug_report_url}>{bug_report_url}</a></td>
-                        </tr>
-
-                        <tr>
-                          <td data-l10n-id="discourse">Discourse</td>
-                          <td><a data-hook="discourse-url" href={discourse_url}>{discourse_url}</a></td>
-                        </tr>
-                      </tbody></table>
-                    </section>}
+                    {!graduated && <div>
```

Comment:
Where does "graduated" come from?  Set somewhere from the graduation field in the yaml?